q2_annotate/busco/busco.py

```diff
@@ -286,24 +331,41 @@ def evaluate_busco(
     )
 
     kwargs = {
-        k: v for k, v in locals().items() if k not in ["mags", "ctx", "num_partitions"]
+        k: v
+        for k, v in locals().items()
+        if k not in ["mags", "unbinned_contigs", "ctx", "db", "num_partitions"]
     }
-
     _evaluate_busco = ctx.get_action("annotate", "_evaluate_busco")
     collate_busco_results = ctx.get_action("annotate", "collate_busco_results")
     _visualize_busco = ctx.get_action("annotate", "_visualize_busco")
+    _filter_contigs = ctx.get_action("assembly", "filter_contigs")
 
+    # add partition cases for unbinned contigs if bins.type <= SampleData[MAGs]:
     if issubclass(mags.format, MultiMAGSequencesDirFmt):
         partition_action = "partition_sample_data_mags"
     else:
         partition_action = "partition_feature_data_mags"
+        warnings.warn(
```

Comment:
Perhaps this warning should only appear if the user actually provided some contigs, otherwise it may be confusing.